docs/DafnyRef/Attributes.md

```diff
@@ -141,32 +95,96 @@ be added.
    if (F != null) { Repr := Repr + {F} + F.Repr; }
 ```
 
-### 22.1.4. axiom
+### 22.1.2. `{:nativeType}` {#sec-nativetype}
+The `{:nativeType}` attribute may only be used on a ``NewtypeDecl``
+where the base type is an integral type. It can take one of the following
+forms:
+
+* `{:nativeType}` - With no parameters it has no effect and the ``NewtypeDecl``
+have its default behavior which is to choose a native type that can hold any
+value satisfying the constraints, if possible, otherwise BigInteger is used.
+* `{:nativeType true}` - Also gives default ``NewtypeDecl`` behavior,
```

Comment:
To be consistent with other attributes, `{:nativeType}` and `{:nativeType true}` should do the same thing. However, that probably lies outside this PR. For now, it's good that we at least document what the current behavior is.

Reply:
Issue filed.